From 6710c5b0674f8811f7d8fbfc526684e7ed77f765 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rafa=C5=82=20Mi=C5=82ecki?= <rafal@milecki.pl>
Date: Thu, 11 Feb 2021 13:12:32 +0100
Subject: [PATCH] dt-bindings: net: rename BCM4908 Ethernet binding
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Rob pointed out that a normal convention is "brcm,bcm4908-enet" so
update whole binding to match it.

Suggested-by: Rob Herring <robh@kernel.org>
Signed-off-by: Rafał Miłecki <rafal@milecki.pl>
Reviewed-by: Florian Fainelli <f.fainelli@gmail.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 .../net/{brcm,bcm4908enet.yaml => brcm,bcm4908-enet.yaml}   | 6 +++---
 MAINTAINERS                                                 | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)
 rename Documentation/devicetree/bindings/net/{brcm,bcm4908enet.yaml => brcm,bcm4908-enet.yaml} (85%)

diff --git a/Documentation/devicetree/bindings/net/brcm,bcm4908enet.yaml b/Documentation/devicetree/bindings/net/brcm,bcm4908-enet.yaml
similarity index 85%
rename from Documentation/devicetree/bindings/net/brcm,bcm4908enet.yaml
rename to Documentation/devicetree/bindings/net/brcm,bcm4908-enet.yaml
index 5f12f51c5b19..c70f222365c0 100644
--- a/Documentation/devicetree/bindings/net/brcm,bcm4908enet.yaml
+++ b/Documentation/devicetree/bindings/net/brcm,bcm4908-enet.yaml
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0-only OR BSD-2-Clause
 %YAML 1.2
 ---
-$id: http://devicetree.org/schemas/net/brcm,bcm4908enet.yaml#
+$id: http://devicetree.org/schemas/net/brcm,bcm4908-enet.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
 title: Broadcom BCM4908 Ethernet controller
@@ -13,7 +13,7 @@ maintainers:
 
 properties:
   compatible:
-    const: brcm,bcm4908enet
+    const: brcm,bcm4908-enet
 
   reg:
     maxItems: 1
@@ -37,7 +37,7 @@ examples:
     #include <dt-bindings/interrupt-controller/arm-gic.h>
 
     ethernet@80002000 {
-        compatible = "brcm,bcm4908enet";
+        compatible = "brcm,bcm4908-enet";
         reg = <0x80002000 0x1000>;
 
         interrupts = <GIC_SPI 86 IRQ_TYPE_LEVEL_HIGH>;
diff --git a/MAINTAINERS b/MAINTAINERS
index 3914691fb4a6..68fcd0a6416b 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -3450,7 +3450,7 @@ M:	Rafał Miłecki <rafal@milecki.pl>
 M:	bcm-kernel-feedback-list@broadcom.com
 L:	netdev@vger.kernel.org
 S:	Maintained
-F:	Documentation/devicetree/bindings/net/brcm,bcm4908enet.yaml
+F:	Documentation/devicetree/bindings/net/brcm,bcm4908-enet.yaml
 F:	drivers/net/ethernet/broadcom/bcm4908enet.*
 F:	drivers/net/ethernet/broadcom/unimac.h
 
